app/src/main/java/io/github/droidkaigi/confsched2018/presentation/about/AboutThisAppActivity.kt

```diff
@@ -21,8 +20,7 @@ class AboutThisAppActivity : BaseActivity(), HasSupportFragmentInjector {
     @Inject lateinit var drawerMenu: DrawerMenu
 
     private val binding: ActivityAboutThisAppBinding by lazy {
-        DataBindingUtil
-                .setContentView<ActivityAboutThisAppBinding>(this, R.layout.activity_about_this_app)
+        drawerMenu.setContentView<ActivityAboutThisAppBinding>(R.layout.activity_about_this_app)
```

Comment:
Hmm, this `setContentView` method name made me confused at first..
I don't have any good idea but I think it's better to rename this or separate the method.
What do you think? cc: @takahirom

Reply:
I also hesitated... 🤔 
If the method separated, do you have a good idea?

Reply:
Sorry, I don't have good idea 🙇  honestly the previous implementation is easier to understand for me.